gradle.properties

```diff
@@ -1,14 +1,7 @@
-projectGroup=org.grails.plugins
 projectVersion=6.0.0-SNAPSHOT
 grailsVersion=7.0.0-SNAPSHOT
 grailsGradlePluginVersion=7.0.0-SNAPSHOT
-gradlePublishPlugin=1.3.0
+gradlePublishPlugin=2.0.0
 springSecurityCoreVersion=7.0.0-SNAPSHOT
 pac4jVersion=6.0.6
-jackson.version=2.18.0
-
```

Comment:
Why did you choose to move these properties into the build.gradle code? 

IMO it's easier to find them here and it's more DRY

Reply:
I am assuming once the grails bom changes are made that the jackson property won't need defined at all since the bom will define it and it will be up-to-date.  I was going to revisit what needs defined once the bom changes are finalized.